Homepage/homepage.css

```diff
@@ -0,0 +1,413 @@
+@import url(/Utility/utility.css);
+
+.homepage-content {
+    height: 100vh;
+}
+
+
+.brand-name,.brand-info h6, .template h6, .particular-content h4 {
+    font-family: 'Playfair Display', serif;
+}
+
+.template h6 {
+    font-size: 3rem;
+}
+
+.template.one h6 {
+    font-size: 5rem;
+}
+
+.icon {
+    color: var(--color-light-gray);
+    font-size: 2rem;
+}
+
+.inner-icon {
+    color: var(--color-light-gray);
+    font-size: 1.3rem;
+    padding: 1rem;
+}
+
+.input-box button {
+    padding: 1.2rem;
+    outline: none;
+}
+
+button {
```

Comment:
try using classes or ids instead of actual components to avoid overwriting and confusion